indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -193,14 +200,39 @@ public boolean run()
     }
   }
 
-  public static class IndexGeneratorMapper extends HadoopDruidIndexerMapper<BytesWritable, Writable>
+  private static IncrementalIndex makeIncrementalIndex(Bucket theBucket, AggregatorFactory[] aggs, StupidPool bufferPool, HadoopDruidIndexerConfig config)
```

Comment:
are there any changes in this function or did it just get moved around? It might be easier to leave it in place for review and worry about formatting separately.